main.py

```diff
@@ -797,7 +797,8 @@ async def perform_filters(request: PerformFiltersRequest):
         
         # Map of filter names to their pipeline IDs
         filter_map = {
-            "NSFW Filter": "nsfw_filter_pipeline"
```

Comment:
not sure if we need this. please confirm

Reply:
We have the enabled filters coming in the request body. We have to use that string to call the respective pipeline. So this mapping helps with that